### DISCUSSION TITLE
Will the app still be a client or will it remain a viewer?

### DISCUSSION THREAD

**Discussion by mondlicht-und-sterne** [jonjomckay/fritter#75] [Q&A] [answered]
the app short description says it would be a client, the features says it is a viewer,
will the app still be a client or will it remain a viewer?

**Comment by jonjomckay** [comment, marked as answer]
The app will be more of a "frontend" than a client or a viewer, I suppose - in a similar vein to Newpipe. The plan is to be able to interact with Twitter, and retain as many features as possible as a standard Twitter app but without requiring an account, and keeping data private and local.

Currently, the app supports subscriptions, grouping subscriptions, trends, and a fair bunch of standard view functionalities (media, profiles, etc.). More features should be expected in due time 😅

**Reply by Mrcactuseater** [reply]
2 years on does this ideology still stand? Now with the 3.x updates the app has a great UI, and with most clients dying off, this & the Mastodon integration would make it a one stop shop.

Thanks again

**Comment by mondlicht-und-sterne** [comment]
thank you very much for your answer.
